utils/i18nUtils.ts

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable simple-import-sort/imports */
-import { LinkPropertyHref } from 'vue-meta' // eslint-disable-line import/named
-import type { Locale, LocaleObject } from 'nuxt-i18n'
+import LinkPropertyHref from 'vue-meta'
```

Comment:
```suggestion
import { LinkPropertyHref } from 'vue-meta' // eslint-disable-line import/named
```
ここは #6721 にあわせて一旦これでお願いします！

Reply:
対応しました～